### DISCUSSION TITLE
Problem for install BIIGLE on a new machine

### DISCUSSION THREAD

**Discussion by guilvinec** [biigle/core#567] [Q&A]
Hi,

I am trying to install biigle on my computer.
I follow the installation guide.
At step 10, when I run cd .. && docker-compose up -d

I have this error message:
```
Pulling websockets (biigle/websockets-dist:)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling websockets (biigle/websockets-dist:)...
ERROR: pull access denied for biigle/websockets-dist, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

I don't understand this error. Can you please tell me what to do to install biigle correctly on my computer?

Thanks in advance,

**Comment by mzur** [comment]
It looks like the previous step did not succeed. Were there any error messages while/after running `build.sh`? Also, please show the output of `docker images`.
